animesaturn.py

```diff
@@ -0,0 +1,336 @@
+                f.write(crwd)
+                f.close()
+            print("Creo crawljob per %d episodi"%len(self.list_link))
+            for link in self.list_link:
+                sourcehtml = requests.get(link[0]).text
+                source = re.findall("file: \"(.*)\",",sourcehtml)
+                try:
+                    mp4_link = source[0]
+                except IndexError:
+                    mp4_link = ""
+                if self.all_ep[link[0]]=="-1":
+                    download = "%s%s/"%(self.movie_path,self.titolo)
+                else:
+                    download = "%s%s/Season_%s"%(self.download_path,self.titolo,self.all_ep[link[0]])
+                crwd = crwd + '''
+                {
+                text= %s
+                downloadFolder= %s
+                enabled= true
+                autoStart= true
+                autoConfirm= true
+                }
+                '''%(mp4_link,download)
+            with open("%s%s.crawljob"%(self.crawl_path,self.titolo), 'a') as f:
+                print(self.crawl_path)
+                f.write(crwd)
+                f.close()
+            self.list_link.clear()
+
+    def downloader(self):
+        #creo un file vuoto, se presente sovrascrivo
+        self.check_Path(str(self.download_path)) #verifico che path esista
+        print("Rilevati %d episodi"%len(self.list_link))
+        episodes = []
+        while True:
+            try:
+                wantedEps = input("Dammi Range Episodi (1:%d) "%len(self.list_link)).split(":")
+                if("all" in (wantedEps[0].lower())):
+                    start, finish = 1, len(self.list_link)
+                elif(len(wantedEps) == 1):
+                    start, finish = int(wantedEps[0]), int(wantedEps[0])
+                else:
+                    start, finish = int(wantedEps[0]), int(wantedEps[1])
+                if(start > 0 and finish > 0 and finish <= len(self.list_link) and start <= len(self.list_link)):
+                    break
+                print("Invalido!")
+            except ValueError:
+                print("Invalido!")
+
+        for episodedata in self.list_link:
+            if(start <= episodedata[1] <= finish):
+                sourcehtml = requests.get(episodedata[0]).text
+
+                try:
+                    mp4_link =  re.findall("file: \"(.*)\",",sourcehtml)
+                except IndexError:
+                    mp4_link = ""
+                episodes.append(mp4_link)
+        print("\n")
+        for ep in sorted(episodes):
+            self.download(ep[0])
+        self.list_link.clear()
+    #riordino correlati e  selezionato in base alla data di uscita
+
+
+    def download(self,url):
+        print(url)
+        file_name = url.split("/")[-1]
+        self.check_Path(os.path.join(self.download_path,file_name.split("_")[0]))
+
+        with open(os.path.join(self.download_path,file_name.split("_")[0],file_name), "wb") as file:
```

Comment:
```suggestion
        with open(os.path.join(self.download_path,file_name.split("_")[0],file_name), "wb"):
```
Variabile `file` non utilizzata